CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ if(CONFIG_BOARD_NRF52_BSIM)
     $ENV{BSIM_COMPONENTS_PATH}/libRandv2/src/
     src/HW_models/
     src/nrfx/
+    src/nrfx/mdk
```

Comment:
I meant in line 24 , sorry for not being too clear (so that we emulate the include directories the real nrfx exposes to the rest of Zephyr)